### PR TITLE
Feature/1 postgres wiring cli for data loading but in golang

### PR DESCRIPTION
**Pull request by yunu-git**
rust -> golang

### CODE REVIEW

File: hut-finder-utilities/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+## How to compile and run
+From the `hut-finder-utilities` directory...
+```
+go mod tidy
+```
+```
+go build
+```
+```
+./hut-finder-utilities-go initialiseHuts
```
Comment (cosmqc) [inline hut-finder-utilities/README.md:16]: Following the steps, the binary created is called `hut-finder-utilities`

File: hut-finder-api/cmd/server/main.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	"os"
+
+	"github.com/gin-gonic/gin"
+)
+
+func main() {
+	err := config.LoadConfig()
+	if err != nil {
+		log.Fatalf("Could not load config: %v", err)
+	}
```
Comment (cosmqc) [inline hut-finder-api/cmd/server/main.go:21]: Is it possible for something more graceful than a fatal error if .env is not present, some fallback or defaults maybe?
Some docs for setting up .env would also be good.

File: hut-finder-utilities/scripts/README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+# Database Setup Guide
+This guide runs you through the setup process of creating a database.
+This assumes you have `postgres` installed.
```
Comment (cosmqc) [inline hut-finder-utilities/scripts/README.md:3]: A cheeky docker image would be really cool to simplify the setup and keep everything containerised.

**Review by cosmqc — CHANGES_REQUESTED** [review]
Looks good Yunu, thanks. Just cleaning up docs a bit, and making setup easier. 
I can try and put in a Docker image if you want, just shout. Postgres was giving me some issues for some reason, I'll give it another try on Wednesday.

File: hut-finder-utilities/README.md (hunk 1 above)
Comment (yunu-git) [reply hut-finder-utilities/README.md:16]: yeah sorry, that was while i was porting things over and forgot to change, will do that now

**Pull request merged**